### PR TITLE
questionpage

### PR DESCRIPTION
**Pull request by yojana-tripathi**
check this

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-          <Route path="/login" element= {  <Login/> }/>
-          <Route path="/forgotpassword" element= {  <ForgotPassword/> }/>
-          <Route path="/:topic/:subTopic" element= {  <SubTopicQuestion/> }/>
-
-
+        <Route exact path="/" element={<Home />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/forgotpassword" element={<ForgotPassword />} />
+        <Route path="/:topic/:subTopic" element={<SubTopicQuestion />} />
+        <Route path="/question" element={<QuestionSeries />} />"
```
Comment (thakursachin05) [inline src/App.jsx:16]: does this routing make any sense , i mean why it is needed ?

File: src/component/questions/Question.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                <div class="accordion-body">
+                  <strong>Explain It</strong> It is shown by default, until the
+                  collapse plugin adds the appropriate classes that we use to
+                  style each element. These classes control the overall
+                  appearance, as well as the showing and hiding via CSS
+                  transitions. You can modify any of this with custom CSS or
+                  overriding our default variables. It's also worth noting that
+                  just about any HTML can go within the{" "}
+                  <code>.accordion-body</code>, though the transition does limit
+                  overflow.
```
Comment (thakursachin05) [inline src/component/questions/Question.jsx:82]: we are not going to manaully type the explain for any question, it has been provided by the question model, please check the quesition model once.

File: src/component/questions/QuestionSeries.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+      question: "What year was JavaScript launched?",
+      options: ["1996", "1995", "1994", "none of the above"],
+      correctAnswerIndex: 1,
+    },
+    {
+      question: "What does CSS stand for?",
+      options: [
+        "Central Style Sheets",
```
Comment (thakursachin05) [inline src/component/questions/QuestionSeries.jsx:25]: this is wrong question model , we don't have the question model like this

**Review by thakursachin05 — CHANGES_REQUESTED** [review]
Changes need : 
image
1. UI is not correct, width of the question component should be more
2. Question No. is aligned and designed properly

File: src/component/questions/Question.jsx (hunk 2 above)
Comment (yojana-tripathi) [reply src/component/questions/Question.jsx:82]: okay i understand but they are for only testing  nothing else

Comment (thakursachin05) [reply src/component/questions/Question.jsx:82]: mam testing should be like what we are going to get in real scenrio.

Comment (thakursachin05) [reply src/component/questions/Question.jsx:82]: we are never going to type the explanation in html

**Pull request closed without merging**